solvers/BMI/util/inPolyhedron.m

```diff
@@ -0,0 +1,13 @@
+function flag = inPolyhedron(verts,pt)
```

Comment:
how does iris do it?  
> On Jul 15, 2015, at 3:50 PM, Hongkai Dai <notifications@github.com> wrote:
> 
> In solvers/BMI/util/inPolyhedron.m <https://github.com/RobotLocomotion/drake/pull/1177#discussion_r34721046>:
> 
> > @@ -0,0 +1,13 @@
> > +function flag = inPolyhedron(verts,pt)
> We do not have one function in drake/util doing this. But with functions in thirdParty/polytopes that I am going to merge in, it will be trivial to do this check
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/RobotLocomotion/drake/pull/1177/files#r34721046>.
>